pulsar-client-tools/src/main/java/org/apache/pulsar/admin/cli/CmdFunctions.java

```diff
@@ -239,9 +239,9 @@ void processArguments() throws Exception {
         @Parameter(names = "--output-serde-classname", description = "The SerDe class to be used for messages output by the function")
         protected String outputSerdeClassName;
         // for backwards compatibility purposes
-        @Parameter(names = "--functionConfigFile", description = "The path to a YAML config file specifying the function's configuration", hidden = true)
+        @Parameter(names = "--functionConfigFile", description = "The path to a YAML config file specifying the functions configuration", hidden = true)
```

Comment:
"The path to a YAML config file that specifies the configuration of a Pulsar Function"